lib/matplotlib/axes/_axes.py

```diff
@@ -2367,11 +2367,14 @@ def stem(self, *args, **kwargs):
           stem(y, linefmt='b-', markerfmt='bo', basefmt='r-')
           stem(x, y, linefmt='b-', markerfmt='bo', basefmt='r-')
 
-        A stem plot plots vertical lines (using *linefmt*) at each *x*
+        A stem plot, by default, plots vertical lines (using *linefmt*) at each *x*
         location from the baseline to *y*, and places a marker there
         using *markerfmt*.  A horizontal line at 0 is is plotted using
         *basefmt*.
 
+        It is possible to create a vertically-oriented stem plot
+        by providing vertical=True.
```

Comment:
Can you convert this to numpydoc format and enumerate all of the things that are popped out of the kwargs in the docstring?